lua/nvim-tree/renderer/builder.lua

```diff
@@ -209,15 +203,15 @@ function Builder:_build_file(node, padding, git_highlight, git_icons_tbl)
   local col_start = offset + #icon + git_icons_length
   local col_end = col_start + #node.name
 
-  if self.special_map[node.absolute_path] or self.special_map[node.name] then
+  if vim.tbl_contains(self.special_files, node.absolute_path) or vim.tbl_contains(self.special_files, node.name) then
```

Comment:
why not keeping existing implementation ?

Reply:
Do you mean the map of file names to boolean values?

I didn't see any reason for false value, so I made it a list like `window_picker.exclude.filetype`.

Reply:
well there is an issue then. Check out line 125 in the PR diff, special_files is used like a map, but here it is used as an array.

Reply:
Great catch!

`_build_folder` and `_build_file` are now the same, both tested OK.